cmake/modules/falcosecurity-libs.cmake

```diff
@@ -28,15 +28,15 @@ else()
   # In case you want to test against a fork of falcosecurity/libs just pass the variable -
   # ie., `cmake -DFALCOSECURITY_LIBS_REPO=<your-gh-handle>/libs ..` 
   if (NOT FALCOSECURITY_LIBS_REPO)
-    set(FALCOSECURITY_LIBS_REPO "falcosecurity/libs")
+    set(FALCOSECURITY_LIBS_REPO "therealbobo/libs")
```

Comment:
I assume you are just testing things; in case please add back the `wip` status :)

Reply:
Yeap! Exactly! Just for testing purposes :)